README.md

```diff
@@ -38,7 +40,8 @@ We use `dotnet` project templates to easily create new projects. It simplifies t
 ### Uninstalling the template
 1. If you want to uninstall the template, run the following command.
```

Comment:
let add in the template folder for this part

Reply:
fixed the sentence